bigchaindb/crypto.py

```diff
@@ -146,6 +144,3 @@ def generate_key_pair():
 
     return (private_value_base58, public_value_compressed_base58)
 
-
-def hash_data(data):
```

Comment:
Yes, agree also on this point, I'm commenting on the main PR, I'd like to move some stuff form `util` to `crypto`, I just noticed my refactoring was inconsistent on some parts.